test/checks/tables/td-headers-attr.js

```diff
@@ -82,6 +82,15 @@ describe('td-headers-attr', function () {
     node = fixture.querySelector('table');
     assert.isFalse(check.call(checkContext, node));
 
+    fixtureSetup(
+      '<table>' +
+        '  <tr> <td id="hi">hello</td> </tr>' +
+        '  <tr> <td headers="hi">goodbye</td> </tr>' +
+        '</table>'
+    );
+    node = fixture.querySelector('table');
+    assert.isFalse(check.call(checkContext, node));
```

Comment:
Probably good to put this in a separate test. This one test already has more asserts than it should.